### PR TITLE
♻️ Strategies

### PR DESCRIPTION
**Pull request by JuampiRombola**

### CODE REVIEW

File: components/strategies/StrategyCard/index.tsx [hunk 1]
```diff
@@ -1,81 +1,100 @@
...
+        flexDirection="column"
+        p={4}
+        gap={3}
+        justifyContent="space-between"
+        minHeight={286}
+        height="100%"
+      >
+        <Box display="flex" flexDirection="column" gap={2.2}>
+          <Box display="flex" flexWrap="wrap" alignItems="center" gap={1}>
+            {tags?.map((label, i) => <StrategyTag key={i} {...label} />)}
```
Comment (sebipap) [inline components/strategies/StrategyCard/index.tsx:85]: same as above

File: i18n/es/translation.json [hunk 2]
```diff
@@ -466,5 +459,27 @@
...
+  "The Health Factor represents how “safe” your leverage portfolio is, defined as the risk-adjusted proportion of collateral deposited versus the borrowed risk-adjusted amount. A health factor below 1x will be considered with a shortfall and open to liquidation.": "El Factor de Salud representa cuán “seguro“ es tu portafolio con apalancamiento, definido como la proporción ajustada al riesgo del colateral depositado frente al monto ajustado al riesgo que se ha tomado prestado. Un factor de salud por debajo de 1x se considerará con un déficit y estará abierto a la liquidación.",
+  "Token allowance surplus after operation": "Excedente de permisos de token luego de la operación",
+  "up to": "hasta",
+  "Refinance Loans": "Refinancia tus préstamos",
+  "FROM": "Desde",
```
Comment (sebipap) [inline i18n/es/translation.json:466]: should it be capitalized?

File: i18n/es/translation.json [hunk 3]
```diff
@@ -466,5 +459,27 @@
...
+  "Deposit EXA on Extra Finance": "Deposita EXA en Extra Finance",
+  "Deposit EXA on Extra Finance and earn interest on it.": "Deposita EXA en Extra Finance y gana intereses sobre él.",
+  "Go to Extra Finance": "Ir a Extra Finance",
+  "Reduce Exposure": "Reduce tu exposición",
+  "Provide Liquidity on Velodrome": "Proporciona Liquidez en Velodrome",
+  "Provide liquidity to the EXA/wETH pool.": "Proporciona liquidez al pool EXA/wETH.",
+  "Go to Velodrome": "Ir a Velodrome",
+  "Bridge & Swap with Socket": "Bridge & Swap con Socket",
+  "Seamlessly bridge and swap assets to OP Mainnet from many different networks.": "Pase y cambie activos de manera fluida a OP Mainnet desde muchas redes diferentes.",
+  "Cross Network": "Red Cruzada",
```
Comment (sebipap) [inline i18n/es/translation.json:476]: I'm not sure about this, but the translation sounds weird
```suggestion
  "Cross Network": "Entre Redes",
```

File: pages/security/periphery.tsx [hunk 4]
```diff
@@ -26,25 +26,41 @@ const Security: NextPage = () => {
...
       ...(displayNetwork.id === optimism.id
         ? [
+            {
+              name: 'DebtManager.sol',
+              audited: true,
+              description: t(
+                'The DebtManager contract is responsible for the leverage, deleverage, and rollover functionality of the protocol.',
+              ),
+              reports: ['ABDK'],
+              information: [`629 ${t('lines')} (515 ${t('lines of code')}), 24.9 kb`], //TODO change
```
Comment (sebipap) [inline pages/security/periphery.tsx:36]: is this final

File: pages/strategies.tsx [hunk 5]
```diff
@@ -1,44 +1,314 @@
...
+        { prefix: t('Health Factor'), text: hfLabel },
+        { text: t('Advanced'), size: 'small' as const },
+      ],
+      button: (
+        <Button fullWidth variant="contained" onClick={() => startLeverager()}>
+          {t('Deleverage')}
+        </Button>
+      ),
+      isNew: true,
+      soruce: 'exactly' as const,
```
Comment (sebipap) [inline pages/strategies.tsx]: maybe I'm wrong but I think we can use `as const` for the whole `featured` array

File: pages/strategies.tsx [hunk 6]
```diff
@@ -1,44 +1,337 @@
...
+            {t('Rollover')}
+          </Button>
+        ),
+        isNew: true,
+      },
+    ],
+    [hfLabel, lowestBorrowAPR, maxYield, startDebtManager, startLeverager, t],
+  );
+
+  const thirdPartStrategies: (Strategy & { chainId?: number })[] = useMemo(
```
Comment (sebipap) [inline pages/strategies.tsx:215]:
```suggestion
  const thirdPartyStrategies: (Strategy & { chainId?: number })[] = useMemo(
```

**Review by jgalat — DISMISSED** [review]

**Review by franm91 — APPROVED** [review]

**Review by sebipap — DISMISSED** [review]
😄

**Review by cruzdanilo — APPROVED** [review]

**Review by sebipap — APPROVED** [review]

**Review by jgalat — APPROVED** [review]

**Review by franm91 — APPROVED** [review]

**Pull request merged**